migrator/README.md

```diff
@@ -0,0 +1,261 @@
+
+```bash
+CustomRecipes.csv
+
+Old Prefix ; Old Length ; Country Code ;  Old Format    ;  New Format   ; Description
+123        ; 7          ; 1            ;  xx3xxxx       ;  xx99xxxx     ; Custom recipe, replace 3 with 99
+001234     ; 7          ; 00           ;  xxxxxxx       ;  xx5xxxxx     ; Custom recipe, add 5 to third index
+```
+
+The above recipes file can be used in place of internal recipes by specifying in the following way:
+
+```bash
+java -jar migrator-1.0.0-jar-with-dependencies.jar --countryCode=00 --number=0012345 --customRecipe=./CustomRecipes.csv
+
+# Output:
+# Migration of country code +00 phone number(s):
+# Successful migration into: +00512345
+```
+### Creating Custom Recipes.csv Files
+
+A Recipes.csv file holds a table containing migration information for all known recipes. A given “recipe” is a row within the table which holds data needed to migrate a stale number into one that is valid and dialable. 
+
+When creating custom recipe files to use, each recipe row must have a key made up of the prefix for the stale numbers that can be represented in the row as well as the possible length of the numbers associated with the row. There should never be a case where two recipes can be used to renumber one stale number so as a result, a given stale number can only ever match with one row in a Recipes table. Each column in the file must be separated using ```;``` and the table must have the following columns: 
+
+```Prefix (key)``` - the prefix of the possible stale numbers that can be represented by the row
+
+```Length (key)``` - the length, of numbers that can be represented in the row
+
+```Old Format``` - represents what digits in the old format need to change, every digit that needs to stay the same is represented as an ‘x’
+E.g. the 3rd index of a 4 digit number needs to be removed/replaced. This number is always 7 → xx7x
+E.g. an addition needs to happen at the 2nd index of a 4 number, so none of the original digits need to be altered → xxxx
+
+```New Format``` - represents what the new number format should be. ‘X’ values represent digits from the original number that will stay the same.
+E.g. digits ‘98’ need to be added at the start of an originally 4 digit number → 98xxxx
+E.g. no digits need to be added because a removal took place. A 5 digit number has now become a 4 digit number → xxxx
+
+```Description``` - text specifying what the change is in words. This will help easily diagnose issues with recipes quickly when the strings in the ‘Old Format’ or ‘New Format’ columns are incorrect.
+
+## Web Application
+
+Please [click here](https://phone-number-migrator.uc.r.appspot.com/) to view the web application version of the migrator tool.
+
+## Disclaimer
```

Comment:
Let's have a privacy section above this maybe talking about the user data and what we do with it.